lib/std/os/windows.zig

```diff
@@ -2015,21 +2015,6 @@ pub fn unexpectedStatus(status: NTSTATUS) std.os.UnexpectedError {
     return error.Unexpected;
 }
 
-pub fn SetThreadDescription(hThread: HANDLE, lpThreadDescription: LPCWSTR) !void {
-    if (kernel32.SetThreadDescription(hThread, lpThreadDescription) == 0) {
-        switch (kernel32.GetLastError()) {
-            else => |err| return unexpectedError(err),
-        }
-    }
-}
-pub fn GetThreadDescription(hThread: HANDLE, ppszThreadDescription: *LPWSTR) !void {
-    if (kernel32.GetThreadDescription(hThread, ppszThreadDescription) == 0) {
-        switch (kernel32.GetLastError()) {
-            else => |err| return unexpectedError(err),
-        }
-    }
-}
-
```

Comment:
Could these still be kept in the stdlib in case anyone wants to call them? If they're not used, then the kernel32 stubs shouldn't be in the compiled binary given Zig's comptime evaluation on reachable code.

Reply:
The agreed on policy (see #4426) is to have minimal surface area so std works, and the rest should go in libraries. There's a lot of unused code and (refer to andrew's comment) eventually the parts that are not used in std should be removed, such as `*A` functions.

Reply:
Ah, good point